src/test/java/uk/gov/hmcts/reform/draftstore/service/draftservice/ReadManyTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.draftstore.service.draftservice;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.BDDMockito;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.draftstore.domain.DraftList;
+import uk.gov.hmcts.reform.draftstore.service.UserAndService;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Matchers.anyString;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReadManyTest extends BaseTest {
```

Comment:
It may be worth checking if the service called the repo with the right arguments.